translation-client/src/main/java/eu/europeana/api/translation/client/utils/TranslationClientUtils.java

```diff
@@ -46,6 +50,44 @@ public static Function<UriBuilder, URI> buildUrl(String path) {
         };
     }
 
+    /**
+     * Creates LangDetectRequest from LanguageDetectionObj
+     * @param languageDetectionObjs
+     * @return
+     */
+    public static LangDetectRequest createLangDetectRequest(List<LanguageDetectionObj> languageDetectionObjs) {
```

Comment:
this looks more like a factory method which can be implemented directly in the class that fires the request. There is no reason to have this as a static method